app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -745,6 +745,14 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                         mode.finish();
                         return true;
 
+                    case R.id.action_share_qr:
+                        Intent intent = new Intent(getBaseContext(), ShareEntryActivity.class);
+                        intent.putExtra("entries", (ArrayList<VaultEntry>) _selectedEntries);
```

Comment:
This will make the Bundle too large and cause a crash if the user selects a lot of entries with icons. So this should either be a list of GoogleAuthInfo, or a list of entry UUIDs.